src/ansys_sphinx_theme/__init__.py

```diff
@@ -446,7 +448,11 @@ def configure_theme_logo(app: Sphinx):
     if not theme_options.get("logo"):
         theme_options["logo"] = pyansys_logo
 
-    if not isinstance(theme_options.get("logo"), str):
+    if isinstance(theme_options.get("logo"), str) and theme_options.get("logo") not in [
+        "ansys",
+        "pyansys",
+        "no_logo",
+    ]:
```

Comment:
Shouldn't this be part of a separate PR?

Reply:
> Shouldn't this be part of a separate PR?

yes, when i was testing this PR with pymechanical, it was blocking the build. i will revert this change and open another PR